benchmarks/utils/benchmark_ts.sh

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env bash
```

Comment:
could this also be made as a python script like the gen md script? Reason being our audience/customers already understand python and will find the script easier to work with